cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -925,6 +925,12 @@ function Configure-GcePdTools {
 '$modulePath = "K8S_DIR\GetGcePdName.dll"
 Unblock-File $modulePath
 Import-Module -Name $modulePath'.replace('K8S_DIR', ${env:K8S_DIR})
+
+  if (ShouldWrite-File ${env:K8S_DIR}\diskutil.exe) {
```

Comment:
Since this utility is not officially released yet, can you wrap this new code in a block:

```
if (Test-IsTestCluster $kube_env) {
  # new code...
}
```

Test-IsTestCluster should return true for clusters running in our e2e test jobs, and for clusters brought up using our [e2e-up.sh steps](https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/windows/README-GCE-Windows-kube-up.md#2b-create-a-kubernetes-end-to-end-e2e-test-cluster).

Reply:
added. Thanks!